src/Explorer/Menus/CommandBar/CommandBarUtil.tsx

```diff
@@ -233,3 +234,28 @@ export const createConnectionStatus = (container: Explorer, poolId: PoolIdType,
     onRender: () => <ConnectionStatus container={container} poolId={poolId} />,
   };
 };
+
+export function createKeyboardHandlers(allButtons: CommandButtonComponentProps[]): KeyboardHandlerMap {
+  const handlers: KeyboardHandlerMap = {};
+
+  function createHandlers(buttons: CommandButtonComponentProps[]) {
+    buttons.forEach((button) => {
+      if (button.disabled !== true && button.keyboardAction) {
```

Comment:
nit:
```suggestion
      if (!button.disabled && button.keyboardAction) {
```

Reply:
I initially had this but was wary of the impact of `disabled` being undefined. But you're right, it's fine because we want `undefined` to be `false` (as it usually is).